### PR TITLE
added code to support multiboot configuration

### PR DESCRIPTION
**Pull request by haribabug**
added code to support multiboot configuration while packaging the SBL OSContainer

### CODE REVIEW

File: classes/acrn-sblimage.bbclass [hunk 1]
```diff
@@ -62,8 +62,9 @@ python do_acrn_sblimage() {
 python() {
     import re
 
-    multibootPackageDependencyList = re.split(" +", d.getVar('MB_DEPENDENCY').lstrip().rstrip())
-    multibootPackageMcDependencyList = re.split(" +", d.getVar('MB_MC_DEPENDENCY').lstrip().rstrip())
+   currentMultibootConfig = d.getVar('BB_CURRENT_MC')
```
Comment (saininav) [inline classes/acrn-sblimage.bbclass:65]: Why it is required ? Can you explain with some scenario ?
Please check MB_MC_DEPENDENCY variable..
https://github.com/intel/meta-acrn/blob/master-next/docs/slimbootloader.md#optional-other-variables

Comment (haribabug) [reply classes/acrn-sblimage.bbclass:65]: We used as below in the conf/local.conf for vm0 and vm1 multiboot config.  THe scenario is ACRN logical partition mode.
BBMULTICONFIG = "sos vm0 vm1"

and used the 'd.getVar('BB_CURRENT_MC')' to get the current configuration.
wihtout the changes in the 'acrn-sblimage.bbclass' the build is failing while creating the Container.

Let us know if any other way to handle our scenario.

Comment (saininav) [reply classes/acrn-sblimage.bbclass:65]: Patch looks Ok. Can you please document BB_CURRENT_MC  variable too at https://github.com/intel/meta-acrn/blob/master-next/docs/slimbootloader.md#optional-other-variables !

Comment (saininav) [reply classes/acrn-sblimage.bbclass:65]: Also set BB_CURRENT_MC default to empty string.
BB_CURRENT_MC  ?= ""

Comment (haribabug) [reply classes/acrn-sblimage.bbclass:66]: updated the slimbootloader.md and assigned the default string to the BB_CURRENT_MC variable.

**Comment by saininav** [comment]
Please do the  BB_CUURNET_MC default initialized changes in same patch and do force push.

Keep document update patch separate.

File: classes/acrn-sblimage.bbclass [hunk 2]
```diff
@@ -62,8 +63,9 @@ python do_acrn_sblimage() {
...
+   multibootPackageDependencyList = re.split(" +", d.getVar('MB_DEPENDENCY').lstrip().rstrip())
+   multibootPackageMcDependencyList = re.split(" +", d.getVar('MB_MC_DEPENDENCY').lstrip().rstrip())
```
Comment (saininav) [inline classes/acrn-sblimage.bbclass:68]: Fix alignment.

File: docs/slimbootloader.md [hunk 3]
```diff
@@ -100,3 +100,7 @@ bzImage bzImage-5.4.52-linux-intel-acrn-sos EFI loader sbl_os
 
 * PREGENERATED_SIGNING_KEY_SLIMBOOT_KEY_SHA256  
   Set the path to the image signing key. Default is "${TOPDIR}/cert/TestSigningPrivateKey.pem"
+
+* BB_CURRENT_MC
+  Get the Current Configuration using this variable when multiple configuration have specified in conf/local.conf using BBMULTICONFIG variable.
```
Comment (saininav) [inline docs/slimbootloader.md:105]: It should be like

* BB_CURRENT_MC  
  Set multiconfiguration dependencies i.e BB_CURRENT_MC  = "sos"
  mcdependency  = "mc:sos:uos:recipe_name:task_on_which_to_depend"

Ref: https://www.yoctoproject.org/docs/current/mega-manual/mega-manual.html#dev-enabling-multiple-configuration-build-dependencies

Comment (haribabug) [reply docs/slimbootloader.md:105]: we set the multiconfiguration as BBMULTICONFIG = "sos vm0 vm1" and get the current configuration using the BB_CURRENT_MC variable.   Not sure if its the right approach. However, can you provide the right wordings to maintain the consistency with other variables definition in the readme file.

File: classes/acrn-sblimage.bbclass (hunk 2 above)
Comment (haribabug) [reply classes/acrn-sblimage.bbclass]: done

File: docs/slimbootloader.md (hunk 3 above)
Comment (saininav) [reply docs/slimbootloader.md:105]: You are right. I overlooked BB_CURRENT_MC variable.

**Pull request merged**